### PR TITLE
Factorio and STEM

### PR DESCRIPTION
**Pull request by Conor12345**
### Author info
Name: Conor_
Suggested release date:

### CODE REVIEW

File: submissions/career/career.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### The computer science angle
+
+I'm not alone in the experience of Factorio guiding my choices elsewhere in life. So many people I've talked to, including our very own *Therenas - ~~LORD GOD~~ Editor-in-chief of Alt-F4*, have starting playing Factorio, experimented with mods then been forced into learning to program so they can fufill their desire to create their own mods. I would *blindly, and with no evidence* suggest there are a not insignificant number of people who learned to program because of Factorio and other moddable games. Even just playing around with combinators and learning the basic of computer logic provides a new skill for many people, maybe pushing them slightly closer to the ever loved STEM fields.
+
+![Playing to modding Factorio meme](https://cdn.discordapp.com/attachments/603392474458882065/745728165116248144/mod_brain.png)
+*Source: Find creator*
```
Comment (Bilka2) [inline submissions/career/career.md]: Hi :) https://discord.com/channels/139677590393716737/306402592265732098/672169819696791582

Comment (Bilka2) [reply submissions/career/career.md]: To be clear, I can't really remember if I originally made this image. I can't find the original file anymore, the png I have saved is from feb 2020 (after the linked discord message). I don't really care to be credited if it was me.

File: submissions/career/career.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+![Engineering Design process diagram](https://teachengineering.org/Images/edpHub/EDPHub_Graphic.jpg)
+*NOTE TO FUTURE CONOR - NEED to get a royalty free version*
+
+### My story with Factorio
+
+The first game I properly played as a *young buck* was Minecraft. Yes, I know, I know, I am very young. After a while in Vanilla Minecraft, I grew bored with the lack of clear goals and limited item range. I was craving more complexity, and a wider range of possibilities. **ENTER MODS**. Modded Minecraft then took over my life. My pack of choice for a good year or two was [Sky Factory 2](https://atlauncher.com/pack/skyfactory), I dipped into other packs here and there, but Sky Factory had the power to hold my attention. Even now I go back and fire up Minecraft 1.7.10 every year or so and have another run through the pack. For anyone who hasn't played a Sky Factory modpack, it essentially involves a traditional Minecraft skyblock with mods which allow you extract resources (such as ores or seeds) from dirt, gravel, sand etc. Most importantly though, this process is automatable.
```
Comment (heinwessels) [inline submissions/career/career.md]: > Even now I go back and fire up Minecraft 1.7.10 every year or so

Saying the same thing two times twice.

File: submissions/career/career.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+*Here, cobblestone is being mined automagically, then crushed into gravel before being sieved and smeling into precious, precious ores.*
+
+The idea that I could make a thing that does my job for me was amazing. Anyone who's played Factorio will of course recognize this idea as the key goal in the game - **Automate *EVERYTHING***. After spending far, far too much time in Sky Factory, I expanded my horizons to other games but found the types of games my friends we're into (FPSes primarily), just weren't my cup of tea. Out in the distance haze, Factorio beckoned me, and I of course bought the game, excited at the possibilities it provided me. 
+
+Unfortunately, it wasn't all sunshine and roses in my journey into the depths of Factorio. As is not uncommon, I gleefully enter the game, made and automated the red and green packs then burned out. Looking back, I suspect there was some Complexity Bias going on here, which again is very common in Factorio if you aren't prepared to break down problems and understand their individual components. The game sat by the wayside, I tried other games and left poor Factorio out in the cold, not a friend in the world.
```
Comment (heinwessels) [inline submissions/career/career.md]: > I tried other games and left poor Factorio out in the cold, not a friend in the world.

Blasphemy!

(I'm just enjoying your style of writing)

File: submissions/career/career.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+![Playing to modding Factorio meme](https://cdn.discordapp.com/attachments/603392474458882065/745728165116248144/mod_brain.png)
+*Source: Find creator*
+
+### Correlation vs Causation
+
+In writing this piece,  I made a lovely little [post](https://www.reddit.com/r/factorio/comments/n8gpes/factorio_and_stem_careers/) on the r/Factorio subreddit, asking people about their relationship with Factorio and STEM, and how it has inspired/changed their education and careers. As always with these kinds of things, I got some of what I wanted, and some interesting new perspectives I hadn't really considered.
```
Comment (heinwessels) [inline submissions/career/career.md]: This one is also really nice, and I think might be worth an add
https://www.reddit.com/r/factorio/comments/n8gpes/factorio_and_stem_careers/gxj9g1x?utm_source=share&utm_medium=web2x&context=3

File: submissions/career/career.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+- The introduction works great, but imo doesn't really connect to the next paragraph, but ends quite abruptly
```
Comment (heinwessels) [inline submissions/career/career.md]: Past Conor (?) basically said what I was going to say XD

File: submissions/career/career.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+So... why am I telling you this? *Fantastic question, really glad you asked. Very well asked question I might add.* For me it's pretty clear that playing modded Minecraft and Factorio pushed forward my critical thinking and mathematical skills and diverted me down the STEM path of goodness. Now, I'm studying Robotic Engineering and though Factorio isn't the only thing that pushed me down this path, it definitely helped guide me to where I am now.
+
+### The computer science angle
```
Comment (heinwessels) [inline submissions/career/career.md]: A few other ideas for expansion:

- Factorio has created fairly large community software projects that pushed people to learn more than simply coding, etc. Clusterio, Factorio Mod Debugger, etc. Modding is a gateway drug to coding.
- Therenas -> forced to start coding -> learned and mastered -> started wikies etc -> now part of Wube for that exact purpose
- Simple mods are really easy to create and theres many tutorials to follow. But, just like Factorio itself, you can dive super deep into complexity and crazy things - if you want to.
- Combinators are similar to what you learn first in your first programming course. When I started with programming at 16 we played with Excel formulas. The goal was to learn "I need this thing, but how do I get it? What is each step to take to get there? " Factorio even teaches this to you without combinators. I want to a furnace array. For that I need create a mine, belts, and the furnaces. For the mine I need more power and kill those biters. I need a belt factory... It's that same endless loop that hooks you to simply keep on playing. I

File: submissions/career/career.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
+- The introduction works great, but imo doesn't really connect to the next paragraph, but ends quite abruptly
+- Your anecdote feels a bit off, it gets a bit to far from STEM, but it is definitely nice to read and highlights some of the reasons someone might get hooked by it
+ - I feel like the CS part is a bit on the shorter side, as well as the Correlation vs Causation part. Regarding the latter, you could maybe talk a bit more about the younger audience, there are definitely people that grew up with Minecraft (might be a bit difficult though, as many people post 2000 are still in school)
+- No conclusion
```
Comment (heinwessels) [inline submissions/career/career.md]: Could mention here that poeple have even written official sciency papers based on Factorio
https://www.reddit.com/r/factorio/comments/lgkcmy/the_factory_must_grow_automation_in_factorio/
https://arxiv.org/abs/2102.04871

File: submissions/career/career.md (hunk 5 above)
Comment (Conor12345) [reply submissions/career/career.md]: Lol this was nano's feedback which I didn't wanna lose at the time :p

File: submissions/career/career.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+*Here, cobblestone is being mined automagically, then crushed into gravel before being sieved and smelting into precious, precious ores.*
+
+The idea that I could make a thing that does my job for me was amazing. Anyone who's played Factorio will of course recognize this idea as the key goal in the game - **Automate *EVERYTHING***. After spending far, far too much time in Sky Factory, I expanded my horizons to other games but found the types of games my friends were into (FPSes primarily), just weren't my cup of tea. Out in the distance haze, Factorio beckoned me, and I of course bought the game, excited at the possibilities it provided me. 
+
+Unfortunately, it wasn't all sunshine and roses in my journey into the depths of Factorio. As is not uncommon, I gleefully enter the game, made and automated the red and green packs then burned out. Looking back, I suspect some Complexity Bias was going on, which again is very common in Factorio if you aren't prepared to break down problems and understand their individual components. The game sat by the wayside, I tried other games and left poor Factorio out in the cold, not a friend in the world.
```
Comment (heinwessels) [inline submissions/career/career.md]: Complexity Bias needs a link. I might need to be better read, but I had to google the term :P

File: submissions/career/career.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+> I applied for a job in IT integration at 36 years old, without training, certificates, experience or anything else relevant. I showed them this topic of a nicely pointless overcomplication of smelting: https://forums.factorio.com/viewtopic.php?p=319620#p319620 I still work there :)
+*u/TheXtrafresh - https://www.reddit.com/r/factorio/comments/n8gpes/factorio_and_stem_careers/gxj9g1x?utm_source=share&utm_medium=web2x&context=3*
+
+## Conclusion
+
+The key parts of Factorio and other [logistics, base building, sandbox, puzzle, problem-solving, real-time strategy](https://www.reddit.com/r/factorio/comments/9z8x5m/does_factorio_merit_the_creation_of_a_new/) games are:
```
Comment (heinwessels) [inline submissions/career/career.md]: This is still a little long-winded. I don't think a conclusion is the right place for a bulleted list of info :P You can maybe rework it to be in a single sentence, and then you reference only a few of those points.

File: submissions/career/career.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+*u/Bigtallanddopey - https://www.reddit.com/r/factorio/comments/n8gpes/factorio_and_stem_careers/gxiwjwy?utm_source=share&utm_medium=web2x&context=3*
+
+> I think it's a correlation. People with STEM aspirations tend to enjoy/be attracted more to games like Factorio. The mindset is typically already there, or else they don't like it and stop playing it.
+*u/Vexinimous - https://www.reddit.com/r/factorio/comments/n8gpes/factorio_and_stem_careers/gxif1dj?utm_source=share&utm_medium=web2x&context=3*
+
+These two comments raise very eloquently the idea that Factorio is an *engineer/scientist/mathematician*'s dream come true. An open playground with clearly defined rules to explore, design and create interesting stuff with no upfront material cost, all whilst sat in our comfy chairs. It's an opportunity for people to explore their creativity and stretch and expand that *oh so special* muscle in the brain responsible for critical thinking and design.
+
+### Nerd sniping for jobs...?
```
Comment (heinwessels) [inline submissions/career/career.md]: Shouldn't this fit in the previous section?

**Comment by Nanogamer7** [comment]
**Fact Checking:** 

As you are rather telling a story than presenting a fact based concept, there isn't much to fact check, however I still listed the few minor details I found below

- ✔ SkyFactory 2 is for Minecraft Version `1.7.10` [[curseforge.com](https://www.curseforge.com/minecraft/modpacks/ftb-presents-skyfactory-2-5/files)]
- ✔ Therenas is our Editor-in-Chief [Discord roles] and Wube's "Documentation developer & Writer" [[factorio.com](https://factorio.com/game/about)]
- ✔ All links direct to where they say they are going, but maybe consider adding a link or changing the link for:
  - SkyFactory 2 to the [curseforge](https://www.curseforge.com/minecraft/modpacks/ftb-presents-skyfactory-2-5/files) page, as the ATLauncher doesn't offer direct downloads, and isn't the original mod page either

---

Further notes:
- Maybe we could add another teaser to the genre thing at the end? Although I'm not sure it really fits

---

✔ Correct
❗ Incorrect
❔ Unclear. This can also mean I couldn't find any resources either proving or disproving a claim

Facts and Statement not listed above weren't validated by me; that doesn't mean they are false, it could be something personal (closed-source, etc.) I can't check, or you already provided a source. If you feel like a claim I left out should be validated, please tell me!

**Pull request merged**